app/src/main/res/values/colors.xml

```diff
@@ -25,7 +25,7 @@
     <!-- Dark Theme -->
     <color name="dark_background_color">#222222</color>
     <color name="dark_dialog_background_color">#424242</color>
-    <color name="dark_settings_accent_color">#ff5252</color>
+    <color name="dark_settings_accent_color">#992722</color>
```

Comment:
I think this should not be changed, so that settings buttons / bullet points / checkboxes / ..., overlayed on top of black/grey, have enough contrast. But I'm not sure this is the color causing that behaviour.

Reply:
Yeah your are correct, I've reverted it now